src/frontend/screens/AuthScreen.tsx

```diff
@@ -74,53 +76,42 @@ export const AuthScreen = ({
     }
   }
 
-  return (
-    <ScrollView
-      ref={scrollViewRef}
-      style={{height: '100%', backgroundColor: WHITE}}>
-      <KeyboardAvoidingView behavior="padding" style={styles.container}>
-        <Image source={require('../images/icon_mapeo_pin.png')} />
-        <Text style={[styles.title]}>Mapeo</Text>
-
-        <Text style={[{marginBottom: 20, fontSize: 16}]}>
-          <FormattedMessage {...m.enterPass} />
-        </Text>
-
-        <PasscodeInput
-          error={error}
-          inputValue={inputtedPass}
-          onChangeTextWithValidation={setInputWithValidation}
-        />
+  const {top} = useSafeAreaInsets();
 
-        {error && (
-          <Text style={[styles.wrongPass]}>
-            <FormattedMessage {...m.wrongPass} />
-          </Text>
-        )}
-      </KeyboardAvoidingView>
-    </ScrollView>
+  return (
+    <ScreenContentWithDock
+      contentContainerStyle={[styles.contentContainer, {paddingTop: top}]}
+      dockContent={
+        error && <Text style={styles.wrongPass}>{t(m.wrongPass)}</Text>
+      }>
+      <Image source={require('../images/icon_mapeo_pin.png')} />
```

Comment:
This should now be the CoMapeo logo!

Reply:
addressed via https://github.com/digidem/comapeo-mobile/pull/503/commits/3cdbf5089f64437e9956714a8dbf553974a42bd7